Orion.Net.Client/Scripts/BaseClientScript.cs

```diff
@@ -91,11 +92,11 @@ protected async Task SendImageContent(string pathImage)
             //Check if file can be read
             try
             {
-                var fileStream = new System.IO.FileStream(pathImage, System.IO.FileMode.Open);
+                var fileStream = new FileStream(pathImage, FileMode.Open);
```

Comment:
This try/catch is useless.
Remove this code, and add a try/catch around the next bloc : var result = ...